.pre-commit-config.yaml

```diff
@@ -3,21 +3,38 @@ repos:
         rev: 5.0.7
         hooks:
               - id: isort
+                alias: isort-cudf
+                name: isort-cudf
+                args: ["--settings-path=python/cudf/setup.cfg"]
```

Comment:
For whatever reason, this hook resorts `__init__.py` files, despite the config specifying to skip them; trying to figure out a way to actually skip them.